packages/core/src/configuration/DefaultCredentialProvider.ts

```diff
@@ -0,0 +1,17 @@
+import { CredentialProvider, Credentials } from './CredentialsProvider';
+import { injectable } from 'tsyringe';
+import { env } from 'process';
+
+@injectable()
```

Comment:
It is not an injectable class. The end-user should be able to create an instance when they want.